cmd/cloudFoundryDeploy_test.go

```diff
@@ -1216,7 +1216,8 @@ func TestDefaultManifestVariableFilesHandling(t *testing.T) {
 func TestExtensionDescriptorsWithMinusE(t *testing.T) {
 
 	t.Run("ExtensionDescriptorsWithMinusE", func(t *testing.T) {
-		extDesc := handleMtaExtensionDescriptors("-e 1.yaml -e 2.yaml")
+		extDesc, extFiles := handleMtaExtensionDescriptors("-e 1.yaml -e 2.yaml")
```

Comment:
Why not `extDesc, _ := ...`?

Reply:
Leftover. Adjusted ... Thanks for pointing to it.

Reply:
I've made the comment only once, but there are four or so more instances of this pattern. Sorry.

Reply:
... ups. Adjusted two more occurances, sorry.